demos/common/cpp/models/src/detection_model_ssd.cpp

```diff
@@ -77,6 +84,40 @@ std::unique_ptr<ResultBase> ModelSSD::postprocess(InferenceResult& infResult) {
     return retVal;
 }
 
+std::unique_ptr<ResultBase> ModelSSD::postprocessMultipleOutputs(InferenceResult& infResult) {
+    LockedMemory<const void> boxesMapped = infResult.outputsData[outputsNames[0]]->rmap();
+    LockedMemory<const void> labelsMapped = infResult.outputsData[outputsNames[1]]->rmap();
+    const float *boxes = boxesMapped.as<float*>();
+    const float *labels = labelsMapped.as<float*>();
+
+    DetectionResult* result = new DetectionResult;
+    auto retVal = std::unique_ptr<ResultBase>(result);
+
+    *static_cast<ResultBase*>(result) = static_cast<ResultBase&>(infResult);
+
+    const auto& internalData = infResult.internalModelData->asRef<InternalImageModelData>();
+
+    for (size_t i = 0; i < maxProposalCount; i++) {
+        float confidence = boxes[i * objectSize + 4];
+        if (confidence > confidenceThreshold) {
+            DetectedObject desc;
+
+            desc.confidence = confidence;
+            desc.labelID = static_cast<int>(labels[i]);
+            desc.label = getLabelName(desc.labelID);
+            desc.x = boxes[i * objectSize] /netInputWidth * internalData.inputImgWidth;
+            desc.y = boxes[i * objectSize + 1] /netInputHeight * internalData.inputImgHeight;
+            desc.width = boxes[i * objectSize + 2] / netInputWidth * internalData.inputImgWidth - desc.x;
+            desc.height = boxes[i * objectSize + 3] / netInputHeight * internalData.inputImgHeight - desc.y;
+
+            /** Filtering out objects with confidence < confidence_threshold probability **/
```

Comment:
It should be near to `if (confidence > confidenceThreshold)`

Reply:
agreed, will fix in next commit